coops-ui/package.json

```diff
@@ -12,27 +12,27 @@
     "test:unit-silent": "vue-cli-service test:unit --silent"
   },
   "dependencies": {
-    "@types/vuelidate": "^0.7.4",
     "axios": "^0.18.0",
     "core-js": "^3.1.4",
     "http-status-codes": "^1.3.2",
     "lodash.isempty": "^4.4.0",
     "regenerator-runtime": "^0.13.3",
     "register-service-worker": "^1.6.2",
-    "sbc-common-components": "1.0.11",
+    "sbc-common-components": "1.0.13",
```

Comment:
I'm confused why this commit is in this PR, since these package.json changes actually went in here (https://github.com/bcgov/lear/pull/356/files).